src/request-factory.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var ResultList = require('./result-list');
+
+function RequestFactory(retriever) {
+  this._retriever = retriever;
+}
+
+RequestFactory.prototype.get = function(options) {
+  return this.resultList(options).next();
```

Comment:
This is not what I meant. I meant that this should delegate to the retriever. Delegating to the resultList doesn't make any sense. We don't even know if it is defined.